### ISSUE TITLE
Web theme should follow UI day/night color like before

### ISSUE THREAD

**Issue by kaetemi** [ryzom/ryzomcore#644]
Before:

![image](https://user-images.githubusercontent.com/1581053/133878363-5948b998-9ed8-475b-b319-388b40d8eae7.png)

Current:

![image](https://user-images.githubusercontent.com/1581053/133878359-4b19e260-8954-4a2d-9d8b-cc04a17dd089.png)

```
nimetu> in css "-ryzom-modulate-color: true;" should work, and global_color attribute on input/img elements like before
```

**Comment by nimetu** [comment]
I got old webig running and I think this should be correct result?
![image](https://user-images.githubusercontent.com/107008/133890016-137b7299-3a4e-4278-93b0-051cd9396300.png)

Well, either case I need to add '-ryzom-modulate-text-color' for text.

browser.css would need this then
```
* { -ryzom-modulate-color: true; }
img, input, a { -ryzom-modulate-color: false; }
```

**Comment by kaetemi** [comment]
Yup, except the darker background of the table cells should match the page background.

**Comment by nimetu** [comment]
I thing I know what you mean, but I dont think its possible without manually changing table/td background to semi transparent.

First half in here has tr bgcolor as #333333d0 and #666666d0
![image](https://user-images.githubusercontent.com/107008/133892926-faff5724-1701-4dcf-acca-a5fc51db81d3.png)

**Comment by kaetemi** [comment]
Hmm. Not sure?

Opening the web in a 2006 client looks like this:
The cells match the background exactly. Not sure why it looks like a different shade in the current browser.

![image](https://user-images.githubusercontent.com/1581053/133917290-e0a29da2-3b99-409e-b078-f910ffda69a2.png)

I'll check again with the latest changes.

**Comment by kaetemi** [comment]
This is with regular browser css

![image](https://user-images.githubusercontent.com/1581053/133919509-09bd119f-3c14-4b09-bac1-acca1b19d9b6.png)

This is with the changes in browser css

```
* { -ryzom-modulate-color: true; }
img, input, a { -ryzom-modulate-color: false; }
```

![image](https://user-images.githubusercontent.com/1581053/133919515-89ff612e-a9aa-495c-82e1-bca267b0f8a6.png)

**Comment by kaetemi** [comment]
Does the background and those triangles end up getting modulated twice, by any chance?

**Comment by nimetu** [comment]
Ok I see the issue. If page background color is 333 and tr/td background color is 333, that tr/td should be invisible.
I think the issue is from drawing two overlapping quads with flush call between them.
Borders have same issue.

**Comment by nimetu** [comment]
The issue is from "inner_thin_border" ingame template that also draws center rectangle.
I draw page background, flush, then draw rest of elements from xml.
inner_thin_border ends up between page and table background/borders.
